app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -176,6 +181,8 @@ def parse_image(image, is_public)
     name ||= $1 if location =~ /^(.+?)(\.(image|img))?\.manifest\.xml$/
     name ||= uid
 
+    labels = parse_labels(image.tags)
```

Comment:
Good catch on the section name, I'll update that. The parse_* convention is just an ems refresher convention that I've noticed. I don't think there's any direct connection to the DB.